src/lib.rs

```diff
@@ -1,7 +1,8 @@
 use chrono::prelude::*;
 use chrono::DateTime;
-
+use sgp4_sys::ClassicalOrbitalElements;
 use thiserror::Error;
+use uom::si::{angle::radian, f64::*, length::kilometer};
 
 mod sgp4_sys;
```

Comment:
@spire-ffoston note that this module is private, so even `pub` types in it are not exported.